### PR TITLE
Threating Shout Nerf

### PR DESCRIPTION
**Pull request by Polisvh**
## Describe your changes
- Change radius to 1.5 from 3
- Change Vulnerability level to 1 from 2
## Checklist before requesting a review
- [+] I have performed a self-review of my code
- [+] I have tested my changes.

### CODE REVIEW

File: champions/src/main/resources/configs/skills/skills.yml [hunk 1]
```diff
@@ -644,11 +644,11 @@ skills:
       enabled: true
       maxlevel: 5
       cooldown: 20.0
-      radius: 3.0
+      radius: 1.5
       cooldownDecreasePerLevel: 1.0
       baseDuration: 3.0
       durationIncreasePerLevel: 1.0
-      vulnerabilityStrength: 2
+      vulnerabilityStrength: 1
```
Comment (Mykindos) [inline champions/src/main/resources/configs/skills/skills.yml:651]: dont double nerf it

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/resources/configs/skills/skills.yml (hunk 1 above)
Comment (Polisvh) [reply champions/src/main/resources/configs/skills/skills.yml:651]: Well well well I can make it vulnerability 2 again though we'll probably still have complaints

Comment (Polisvh) [reply champions/src/main/resources/configs/skills/skills.yml:651]: This shit already deals 9 damage and also boosts outcoming damage for next 7 seconds,  and can be applied from 15 blocks away dude

Comment (Polisvh) [reply champions/src/main/resources/configs/skills/skills.yml]: I've made it back to vuln 2

**Pull request merged**